.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Report a bug, false-positive or false-negative
+title: '[BUG] '
+labels: ''
+assignees: ''
+
+---
+
+## Describe the bug
+
+<!-- A clear and concise description of what the bug is. -->
+
+## Steps To Reproduce
+
+<!--
+Provide the steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+-->
+
+## Analyzer
+
+**Package**: [Microsoft.CodeAnalysis.FxCopAnalyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.FxCopAnalyzers)
```

Comment:
💭 I always found it difficult to answer this, and it's going to get harder with .NET 5. Not sure how to fix it though.

Reply:
Maybe that's worth removing (at least for now) and see later on if we are able to add something helping you?

Reply:
Ping @sharwell

Reply:
Lets keep this package in the template for now. We can remove it later once we ship analyzers with .NET SDK.

Reply:
Ok!